src/pages/inventory/CreateBrand.tsx

```diff
@@ -57,7 +57,7 @@ const CreateBrand = () => {
           setLoading(false);
           setAlert({
             severity: 'error',
-            message: `Error creating supplier: ${err.message}. Try again later.`
+            message: `Error creating supplier! Try again later.`
```

Comment:
can see src/pages/inventory/CreateProduct for example of how to get the specific error msg from BE